python/utils.py

```diff
@@ -9,18 +9,19 @@ class zellowork_api():
     Function class for python zellowork api
     '''
 
-    def __init__(self, api_key, network):
+    def __init__(self, api_key, network, base_domain="zellowork.com", verifyTls=True):
```

Comment:
This new parameter doesn't follow the snake case convention found in the rest of the file.

Reply:
Fixed. Good catch!